DEVELOPER.md

```diff
@@ -375,8 +375,12 @@ Our Jupyter demo at https://demo.kamu.dev includes a special Jupyter notebook im
 4. Run `make minio-data` - this will prepare example datasets to be included into `minio` image
 5. Prepare your `docker buildx` to build multi-platform images (see [instructions below](#building-multi-platform-images))
 6. Run `make minio-multi-arch` to build **and push** multi-arch `minio` image
-7. Run `make jupyter-multi-arch` to build **and push** multi-arch `jupyter` image
-8. You can now proceed to deploy the new image to kubernetes environment
+7. Setup github access token:
+  7.1. Go to [Github access token page](https://github.com/settings/tokens/new) and generate 
```

Comment:
How about preparing the link to reduce actions number?

For example:
https://github.com/settings/tokens/new?description=Upload%20Kamu%20packages&scopes=write%3Apackages